omero/sysadmins/unix/server-postgresql.txt

```diff
@@ -25,7 +25,7 @@ the following commands:
        $ createdb -V
        createdb (PostgreSQL) 9.3q.4
```

Comment:
I guess these code examples are okay as they are illustrating how to find the version. Does `9.3q.4` exist? Just wondering if the three version strings should be the same?